lib/fungus_toast/games/helpers/player_skills.ex

```diff
@@ -0,0 +1,162 @@
+defmodule FungusToast.PlayerSkills do
+  @moduledoc """
+  The PlayerSkills helper for the Games context.
+  """
+
+  import Ecto.Query, warn: false
+  alias FungusToast.Repo
+
+  alias FungusToast.Games
+  alias FungusToast.Games.{Player, PlayerSkill, Skill}
+
+  @doc """
+  Returns the skill information for a given player.
+
+  ## Examples
+
+      iex> get_player_skill(player, 1)
+      %PlayerSkill{}
+
+      iex> get_player_skill(1, 1)
+      %Player{}
```

Comment:
Is this correct? The doc here states that we'll return 2 different types dependent on the type we pass in to the function. Maybe I missed it, but I didn't see a `doctest` for this module so I'm not sure I trust the doc correctness.

Reply:
It should be %PlayerSkill{}